addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -459,6 +459,36 @@ def test_assign_quota(self):
             "Assignment quota: no compensation and no limit anymore (do as asked even with 30 already assigned)"
         )
 
+    # test to mitigate random.choice from raising the zero weight validation error
```

Comment:
This should be the test docstring, under the function definition like `"""Docstring string."""`

Reply:
Done!